cppcache/src/ThinClientLocatorHelper.hpp

```diff
@@ -71,6 +74,8 @@ class ThinClientLocatorHelper {
   const ThinClientPoolDM* m_poolDM;
   ThinClientLocatorHelper(const ThinClientLocatorHelper&);
   ThinClientLocatorHelper& operator=(const ThinClientLocatorHelper&);
+  std::string m_sniProxyHost;
```

Comment:
If the set of member variable is small perhaps its time to make them conform to the style guide. (optional)

Reply:
Changes coming to fix this.  We changed the new members to conform to style guide after rebasing on top of your "cryptoimpl removal" change.